go/vt/vttablet/tabletserver/query_executor.go.orig

```diff
@@ -0,0 +1,939 @@
+/*
```

Comment:
looks like this file got accidentally pushed.

Reply:
:facepalm:

Reply:
removed